OpenRA.Mods.Common/Effects/Bullet.cs

```diff
@@ -31,8 +31,8 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Image to display.")]
 		public readonly string Image = null;
 
-		[Desc("Loop this sequence of Image while this projectile is moving.")]
-		[SequenceReference("Image")] public readonly string Sequence = "idle";
+		[Desc("Loop these sequences of Image while this projectile is moving.")]
+		[SequenceReference("Image")] public readonly string[] Sequences = { "idle" };
```

Comment:
Yeah, I didn't have any test cases that broke, so I figured that was handled automatically by reflection. I'll write an upgrade rule.